db/citations.yml

```diff
@@ -8566,11 +8566,11 @@ nanarong;npj24:
   doi: 10.48550/arXiv.2410.17467
   entrytype: article
   facility: ''
-  grant: TRI
-  journal: ''
+  grant: tri22
+  journal: arXiv
   month: ''
   nb: ''
-  note: submitted to npj Comput. Mater.
+  note: https://doi.org/10.48550/arXiv.2410.17467
```

Comment:
Since this is a URL it can go in the url field.

Here we want to put text so the reference will be correctly generated, so look in arXiv for how to reference this, and then write here the words.  Bear in mind that since we wrote arxiv in the journal, it will reproduce this as the journal.

If you want to see how it will compile you can run `regolith build publist --PEOPLE tnanarong --no-pdf`  It will build your publication list and you can go to `_build/publist` and find the tex file nad bib file and run latex to build your publlist and see what it looks like.